src/librustdoc/html/markdown.rs

```diff
@@ -115,15 +116,19 @@ macro_rules! event_loop_break {
             match event {
                 $($end_event)|* => break,
                 Event::Text(ref s) => {
+                    debug!("Text");
```

Comment:
I'm not opposed to keeping the debugging statements, but it'd be nicer if there were a little more descriptive, maybe something like `debug!("markdown parser encountered 'Text'")`, whatever works though.